modules/security-overview.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='security-overview-{context}']
+= Security in {product-title}
+
+The {product-title} and Kubernetes APIs authenticate users who present
```

Comment:
This is not a great set of content (it’s really unbalanced).  The auth teams need to provide feedback to correct this but it should be talking about our threat model, the different tools we have for security, and provide high level overview of our security subsystems.  This page as it is is not useful.